src/test/java/io/odh/test/e2e/standard/UninstallST.java

```diff
@@ -38,12 +40,13 @@
         @Step(value = "Deploy DSC", expected = "DSC is created and ready")
     }
 )
-@Disabled("Disabled because of the https://issues.redhat.com/browse/RHOAIENG-499")
+@DisabledIf(value = "isOdhTested", disabledReason = "This test needs to be modified to work on ODH.")
```

Comment:
More like "ODH needs to be modified to work with the test" ;P

The problem isn't ODH but OLM, no? If you install with bundle, operator cannot uninstall, but if you install with OLM, the uninstall code (should) work on ODH as well.

Reply:
I'm not sure yet, I have to check with a real installation.